CONTRIBUTING.md

```diff
@@ -31,14 +31,14 @@ Navigate to the project folder and install necessary dependencies:
 
 ```shell
 cd jackson
-npm run custom-install
+npm install
```

Comment:
`npm run custom-install` is correct, we have some legacy peer dependencies which fail with `npm install`

Reply:
This is my first Pull request i have ever made please ignore all this mistakes sir